dreadnode/cli/platform/tag.py

```diff
@@ -0,0 +1,40 @@
+import platform
```

Comment:
These functions could be in compose or version, but the cyclic issues pushed me to segment it.

I do wonder longterm about making Tag a structured object with some parsing and helpers so we aren't always doing translations.